.github/workflows/selfbuild.yml

```diff
@@ -25,6 +25,8 @@ jobs:
             os: ubuntu-20.04
           - target_arch: aarch64-unknown-linux-gnu
             os: ubuntu-20.04
+          - target_arch: aarch64-unknown-linux-musl
+            os: ubuntu-20.04
```

Comment:
You forgot to also update .github/workflows/build-package.yml.template

Reply:
What do I need to change?

Reply:
You need to also add this target to that file

Reply:
I didn't have to do that in my previous PR.